lldb/test/API/lang/swift/po/objc/Makefile

```diff
@@ -0,0 +1,21 @@
+OBJC_SOURCES := main.m
+CFLAGS_EXTRAS = $(MANDATORY_MODULE_BUILD_CFLAGS) -I. -I$(BUILDDIR)
+
+SWIFT_SOURCES := Foo.swift
+SWIFTFLAGS_EXTRAS = -Xcc -I$(SRCDIR) -emit-objc-header-path Foo.h -parse-as-library
+
+#LD_EXTRAS = -lFoo -L$(BUILDDIR)
+
+all: Foo.swift.o $(EXE)
+
+include Makefile.rules
+
+#lib%.dylib: %.swift
```

Comment:
This is all commented out

Reply:
Leftover from a more complicated version of this test.